theme/templates/accounts/profile.html

```diff
@@ -265,38 +265,39 @@ <h3 class="panel-title"><span class="glyphicon glyphicon-phone-alt">&nbsp;</span
                                         <td><span>Generic</span></td>
                                         <td><span class="badge">0</span></td>
                                     </tr>
+                                     <tr id="badge-geographic-feature" data-type="type-Geographic Feature (ESRI Shapefiles)">
+                                        <td><span>Geographic Feature</span></td>
+                                        <td><span class="badge">0</span></td>
+                                    </tr>
                                     <tr id="badge-geographic" data-type="type-Geographic Raster">
                                         <td><span>Geographic Raster</span></td>
                                         <td><span class="badge">0</span></td>
                                     </tr>
-                                    <tr id="badge-his" data-type="type-HIS Referenced Time Series">
-                                        <td><span>HIS Referenced Time Series</span></td>
+{#                                    <tr id="badge-his" data-type="type-HIS Referenced Time Series">#}
+{#                                        <td><span>HIS Referenced Time Series</span></td>#}
+{#                                        <td><span class="badge">0</span></td>#}
+{#                                    </tr>#}
```

Comment:
I don't know if I'm missing something, but in my instance of the develop branch this type of resources no longer exists, so I disabled the filtering option for it.